Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.hpp

```diff
@@ -0,0 +1,134 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Extent.hpp"
+#include "Acts/Seeding/BinnedGroup.hpp"
+#include "Acts/Seeding/InternalSpacePoint.hpp"
+#include "Acts/Seeding/SeedFinderConfig.hpp"
+#include "Acts/Utilities/Grid.hpp"
+
+#include <vector>
+
+namespace Acts {
+
+/// Cylindrical Space Point bin is a 2D grid with (phi, z) bins
+/// It stores a vector of internal space points to external space points
+template <typename external_spacepoint_t>
+using CylindricalSpacePointGrid =
+    Acts::Grid<std::vector<std::unique_ptr<
+                   Acts::InternalSpacePoint<external_spacepoint_t>>>,
+               Acts::detail::Axis<Acts::detail::AxisType::Equidistant,
+                                  Acts::detail::AxisBoundaryType::Closed>,
+               Acts::detail::Axis<Acts::detail::AxisType::Variable,
+                                  Acts::detail::AxisBoundaryType::Bound>>;
+
+/// Cylindrical Binned Group
+template <typename external_spacepoint_t>
+using CylindricalBinnedGroup =
+    Acts::BinnedGroup<Acts::CylindricalSpacePointGrid<external_spacepoint_t>>;
+
+template <typename external_spacepoint_t>
+using CylindricalBinnedGroupIterator = Acts::BinnedGroupIterator<
+    Acts::CylindricalSpacePointGrid<external_spacepoint_t>>;
+
+struct CylindricalSpacePointGridConfig {
```

Comment:
The actual properties here are unchanged, right?

Reply:
right, it's just renaming